roles/disk/README.rst

```diff
@@ -0,0 +1,9 @@
+Disk
+======
+
+.. versionadded:: 1.3
+
+The disks role simply configures a ``mantldata`` logical volume and allocates
+the remainder (by default) of the mantl physical volume to ``/mnt/mantldata``.
```

Comment:
Where is the physical volume mounted?

What if applications that we haven't configured to use this new disk attempt to store more on the physical volume?